src/main/java/CycleTimeGUI.java

```diff
@@ -0,0 +1,73 @@
+import com.fasterxml.jackson.databind.JsonNode;
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.chart.Axis;
+import javafx.scene.chart.ScatterChart;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.VBox;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+
+
+import java.util.List;
+
+public class CycleTimeGUI extends Application {
+
+    private int projectID;
+    private String authToken;
+    private String TAIGA_API_ENDPOINT;
+    private String firstDate;
+    private String lastDate;
+
+    public CycleTimeGUI(int projectID,String authToken, String TAIGA_API_ENDPOINT,String firstDate, String lastDate) {
```

Comment:
instead of taking these details here, why not call CycleTime in DisplayPage and only pass the metrics map?